client/src/ui/css/App.css

```diff
@@ -35,6 +35,82 @@
   background-position-y: 103%;
 }
 
+.background-gradient_afternoon{
```

Comment:
Please factor out the common css style here -- for example background-repeat, background-size, etc.